airgun/entities/rhsso_login.py

```diff
@@ -11,10 +13,23 @@ def login(self, values):
         view.fill(values)
         view.submit.click()
 
+    def login_again(self):
+        view = RhssoExtLogoutView(self.browser)
+        view.login_again.click()
+        return view.read()
```

Comment:
I feel there should be better way to approach that, but I'm not 100% sure without seeing how things work on Robottelo side.

As an idea, could we move logic to `NavigateToLogin` class? So you would have single `RHSSOLoginEntity.login` method that uses `self.navigate_to(self, 'NavigateToLogin')`. But `navigate_to()` would be smart to look where it is and take action based on that. If it is on `RhssoLoginView`, it would do one thing. If it is on `RhssoExtLogoutView`, it would first click `login_again` button.

Take a look at [navmazing documentation](https://github.com/RedHatQE/navmazing) to see if there is any method you could hook into.

As discussed off GitHub, navmazing is not really designed for this kind of circular navigations, so I'll understand if there is nothing we could do. But I think we should at least explore other possibilities.

Reply:
@mirekdlugosz  below code snippet makes sense? 

``` python
    def login(self, values, ext_login=False):
        if not ext_login:
            view = RhssoExtLogoutView(self.browser)
            view.login_again.click()
            return view.read()
        else:
            view = self.navigate_to(self, 'NavigateToLogin')
            view.fill(values)
            view.submit.click()
```